packages/components/src/custom-gradient-picker/style.scss

```diff
@@ -37,20 +34,18 @@ $components-custom-gradient-picker__padding: 6px; // 36px container, 24px handle
 	}
 
 	.components-custom-gradient-picker__control-point-button {
-		border: 2px solid transparent;
-		box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) $white;
+		border: 1.5px solid $white;
```

Comment:
Not sure if we should use sub-pixel widths — it can create rendering glitches

Reply:
We actually use 1.5px for the focus style across the board, but good catch — that is rendered using `box-shadow` so as to actually function, whereas I believe borders are rounded to nearest whole. 

For additional context, we usually pair the box shadow with a transparent border or outline, because in Windows High Contrast mode (which is looking like it might become a web standard), box shadows are removed entirely, but transparent colors are made opaque. So in those modes, the transparent border would work as the focus style.

Reply:
Hi @jasmussen, here we have an issue because we have a 1.5px border and we also need a box-shadow (according to figma). So I'm not sure what we should do it seems hard to have a 1.5 border and also a box-shadow (at least without some complex hack).

Reply:
I took the liberty of pushing a fix in https://github.com/WordPress/gutenberg/pull/34712/commits/2997a2f2b933b6d157862bf7ad50a67411abea79, which I don't think is too hacky. It just stacks multiple box shadows by comma separation.

Reply:
I wonder if we should be using the `CONFIG.borderWidthFocus` variable here for the border width — and if we should change its value to be `2px` instead of `1.5px` (see also related comment https://github.com/WordPress/gutenberg/pull/34598#issuecomment-924030146)

Reply:
We should definitely use variables whenever we can. It's worth noting that we use 1.5px for focus styles across the entire editor, so it's a rather big change. What rendering glitches have you seen?

Reply:
I noticed [this glitch in the Color Picker PR](https://github.com/WordPress/gutenberg/pull/34598#pullrequestreview-753780744), and changing the value to `2px` [seemed to fix it](https://github.com/WordPress/gutenberg/pull/34598#issuecomment-924030146) for me — although I agree with you about this being quite an impactful change that would need a deeper investigation first.

Reply:
But that uses `border`: https://github.com/WordPress/gutenberg/pull/34598/files#diff-172e1f5e636f1d05b7c28a61e4885cd8fddc9dc390952a5547d271dfb2c7663bR94 which is always rounded to nearest whole pixel. The solution I added in https://github.com/WordPress/gutenberg/commit/2997a2f2b933b6d157862bf7ad50a67411abea79 uses `box-shadow`, which renders on subpixels as well.

Reply:
I see, thank you for the explanation! Should we look into applying the `box-shadow` technique to borders across components (including the ColorPicker PR), then?

Reply:
Yes, the benefit of box shadows is that you can stack as many of them as you need, to accomplish very complex designs, such as the 1.5px inner stroke with a subtle outer drop shadow.

The one thing we need to keep in mind, though, is to always pair it with a `transparent` border or outline, as box-shadows are removed entirely in Windows High Contrast mode, whereas transparent colors are made opaque. By combining both, the shadow is shown in normal modes, and the transparent outline or border is shown in high contrast modes. There's a little more nuance in [this video](https://make.wordpress.org/design/2020/10/29/design-show-tell-windows-10-high-contrast-mode/).